pykilosort/gui/main.py

```diff
@@ -0,0 +1,216 @@
+import os
+import numpy as np
+from pathlib import Path
+# TODO: optimize imports before incorporating into codebase
```

Comment:
We can fix this automatically with some tooling later (e.g. `isort` or `black`). dw about it for now.

Reply:
I'll remove the TODO, then (?)